### DISCUSSION TITLE
Content proposal: add BCD data for Deno to MDN compat tables

### DISCUSSION THREAD

**Discussion by lucacasonato** [mdn/mdn-community#540] [Content] [answered]
I am proposing to add web API compatibility information to MDN compat tables, like is done for Node currently. Deno data is in the process of being imported into BCD right now: https://github.com/mdn/browser-compat-data/pull/10753. Some background from [the proposal in the BCD repo](https://github.com/mdn/browser-compat-data/issues/5476):

> Deno aims to be a web browser for scripts. We can run user code in a sandbox similar to web browsers, have the ability to import code in the same way as web browsers, implement the same APIs as web browsers, and use the same test suite as web browsers (WPT). Deno is effectively a web browser without DOM. As a consequence, the first place users will look when figuring out how a Web API works is MDN.
> 
> MDN is also hugely important for discovery of web APIs. If you look up "fetch request javascript" on Google, the first result will be "Using Fetch - Web APIs | MDN". Navigating to that page a user can see what browsers the API is available in. It would be great for Deno users if that chart also showed compatibility in Deno. For some APIs like URL, server side runtimes are already listed (Node), so this would not be a crazy new change.

For all intents and purposes this could be treated very similarly to Node.js information in the compat tables currently. The column would only show up if at least one row in the set includes the data. I would be happy to implement the required changes for this in [yari](https://github.com/mdn/yari) if the suggestion is accepted.

cc @ddbeck

**Comment by ddbeck** [comment]
@Elchi3 and I talked about this in our most-recent 1:1. I think we're both in favor of this. 👍

@lucacasonato has been very response to BCD's process and reviews on the forthcoming data PR, so I expect this addition to the compat table will be high-quality and cover the features that are relevant to JS developers using Deno. Since handling the data will be quite similar to Node.js, I don't expect much risk in going forward on this.

**Reply by Rumyra** [reply]
Adding a note so i can track this convo 👍

**Comment by ddbeck** [comment, marked as answer]
The discussion for this fragmented a little, but to properly conclude this discussion: Deno got the OK from the MDN team in August. It was accepted for compat tables and the work was completed in the course of https://github.com/mdn/yari/pull/4451. It seems to be working well! Thank you, @lucacasonato!

**Comment by pepelsbey** [comment]
Closing as answered/settled.
